src/Microsoft.Health.Fhir.Core/Features/Validation/ResourceContentValidator.cs

```diff
@@ -3,44 +3,41 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
 using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
 using EnsureThat;
-using FluentValidation;
 using FluentValidation.Results;
 using FluentValidation.Validators;
 using Microsoft.Health.Fhir.Core.Models;
-using Task = System.Threading.Tasks.Task;
 using ValidationResult = System.ComponentModel.DataAnnotations.ValidationResult;
 
 namespace Microsoft.Health.Fhir.Core.Features.Validation
 {
-    public class AttributeValidator : IPropertyValidator
+    /// <summary>
+    /// Validates content of resource.
+    /// </summary>
+    /// <remarks>
+    /// Even if we correctly parsed resource into object it doesn't mean resource is valid.
+    /// We need to check what property filled properly, they have right cardinality, correct types, proper format, etc.
```

Comment:
nit: perhaps shorten to read better:
```suggestion
    /// We need to check that properties have right cardinality, correct types, proper format, etc.
```